sormas-e2e-tests/src/main/resources/configuration/properties/environment/performance.properties

```diff
@@ -0,0 +1,56 @@
+#
+# SORMAS® - Surveillance Outbreak Response Management & Analysis System
+# Copyright © 2016-2021 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+
+#
+# SORMAS® - Surveillance Outbreak Response Management & Analysis System
+# Copyright © 2016-2021 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+
+#
+# SORMAS® - Surveillance Outbreak Response Management & Analysis System
+# Copyright © 2016-2021 Helmholtz-Zentrum für Infektionsforschung GmbH (HZI)
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+#
+
+ENVIRONMENT=performance
+ENVIRONMENT_URL=https://test-performance.sormas.netzlink.com/sormas-ui/login#!cases/?includeCasesFromOtherJurisdictions=false&dateFilterOption=DATE
```

Comment:
Can we configure this from outside the build, or do we have to name all supported instances as environment? Then rename this environment to `test-performance`.

Reply:
I renamed it to `test-performance` and removed the `ENVIRONMENT ` from the file.  You are right this can be provided when the gradle command is executed :
`./gradlew clean -Dcucumber.tags="@Login" -Denvironment=test-performance`